docs/en/concept/schema-feature.md

```diff
@@ -24,6 +27,20 @@ schema = {
 }
 ```
 
+### table
+
+The table full name of the table identifier which the schema belongs to, it contains database, schema, table name. e.g. `database.schema.table`, `database.table`, `table`.
+
+### schemaFirst
+
+Default is false.
+
+If the schemaFirst is true, the schema will be used first, this means if we set `table = "a.b"`, a will be parsed as schema rather than database, then we can support write `table = "schema.table"`.
```

Comment:
```suggestion
If the schema_first is true, the schema will be used first, this means if we set `table = "a.b"`, `a` will be parsed as schema rather than database, then we can support write `table = "schema.table"`.
```

Reply:
done.